### PR TITLE
Add configuration for organization

### PR DESCRIPTION
**Pull request by Vampouille**
Add new values for 'requiredFields' : "orgShortName", "orgAddress" and
"orgType".
Add configuration parameter : 'orgTypeValues' for allowed value of
organization types.

### CODE REVIEW

**Comment by Vampouille** [comment]
After merging this PR, we need merge 'master' branch into 'docker-master' branch.

**Comment by Vampouille** [comment]
Modification of 5400712 is already present in docker-master

File: ldapadmin/ldapadmin.properties [hunk 1]
```diff
@@ -4,8 +4,10 @@ protectedUser.uid1=geoserver_privileged_user
 moderatedSignup=true
 moderatorEmail=georchestra+testadmin@georchestra.mydomain.org
 delayInDays=1
-requiredFields=firstName,surname
+#possible value for org creation form : "orgShortName", "orgAddress" and "orgType"
+requiredFields=firstName,surname,org,orgType
 instanceName=geOrchestra
+orgTypeValues=Association,Entreprise,Organisation non gouvernementale
```
Comment (fvanderbiest) [inline ldapadmin/ldapadmin.properties]: in english ...

File: ldapadmin/ldapadmin.properties [hunk 2]
```diff
@@ -4,8 +4,10 @@ protectedUser.uid1=geoserver_privileged_user
 moderatedSignup=true
 moderatorEmail=georchestra+testadmin@georchestra.mydomain.org
 delayInDays=1
-requiredFields=firstName,surname
+#possible value for org creation form : "orgShortName", "orgAddress" and "orgType"
```
Comment (fvanderbiest) [inline ldapadmin/ldapadmin.properties]: possible values

**Comment by fvanderbiest** [comment]
Minor comments, LGTM otherwise. Thanks !

**Pull request merged**